fireblocks_sdk/api_types.py

```diff
@@ -257,6 +257,24 @@ def __init__(self, name_prefix=None, name_suffix=None, min_amount_threshold=None
         self.before = before
         self.after = after
 
+class PagedExchangeAccountRequestFilters(object):
+    """ Optional filters to apply for request
+
+    Args
+
+        limit (number, optional): Results page size
+        before (string, optional): cursor string received from previous request
+        after (string, optional): cursor string received from previous request
+
+    Constraints
+        - You should only insert 'before' or 'after' (or none of them), but not both
```

Comment:
Instead of relying on the docstring, please add the relevant check inside the constructor